lib/listcoef.gi

```diff
@@ -260,70 +260,99 @@ function( l, s )
 end );
 
 
+# FIXME: Remove the deprecated five argument version method
+# (and also its declaration)
 #############################################################################
 ##
-#M  MultRowVector( <list1>, <poss1>, <list2>, <poss2>, <mult> )
+#M  MultRowVectorLeft( <list1>, <poss1>, <list2>, <poss2>, <mult> )
 ##
-InstallMethod( MultRowVector,"generic method",
+InstallMethod( MultRowVectorLeft, "generic method",
     true,
     [ IsDenseList and IsMutable,
       IsDenseList,
       IsDenseList,
       IsDenseList,
       IsMultiplicativeElement ],
     0,
-
 function( l1, p1, l2, p2, m )
+    Print( "#I  This usage of `MultRowVectorLeft` is no longer supported." );
     l1{p1} := m * l2{p2};
 end );
 
-InstallOtherMethod( MultRowVector,"error if immutable",true,
+InstallOtherMethod( MultRowVectorLeft, "error if immutable", true,
     [ IsList,IsObject,IsObject,IsObject,IsObject],0,
     L1_IMMUTABLE_ERROR);
 
 #############################################################################
 ##
-#M  MultRowVector( <list>, <mul> )
+#M  MultRowVectorLeft( <list>, <mul> )
+#M  MultRowVectorRight( <list>, <mul> )
 ##
-InstallOtherMethod( MultRowVector,
-        "two argument generic method",
+InstallMethod( MultRowVectorLeft,
+    "two argument generic method",
     true,
     [ IsDenseList and IsMutable,
       IsMultiplicativeElement ],
     0,
-
 function( l, m )
     local   i;
-
     for i  in [ 1 .. Length(l) ]  do
         l[i] := m * l[i];
     od;
 end );
 
-InstallOtherMethod( MultRowVector,"error if immutable",true,
-    [ IsList,IsObject],0,
+InstallMethod( MultRowVectorRight,
+    "two argument generic method",
+    true,
+    [ IsDenseList and IsMutable,
+      IsMultiplicativeElement ],
+    0,
+function( l, m )
+    local   i;
+    for i  in [ 1 .. Length(l) ]  do
+        l[i] := l[i] * m;
+    od;
+end );
+
+InstallOtherMethod( MultRowVectorLeft, "error if immutable", true,
+    [ IsList, IsMultiplicativeElement ],
+    0,
+    L1_IMMUTABLE_ERROR);
+
+InstallOtherMethod( MultRowVectorRight, "error if immutable", true,
+    [ IsList, IsMultiplicativeElement ],
+    0,
     L1_IMMUTABLE_ERROR);
 
-InstallOtherMethod( MultRowVector,
-        "Two argument kernel method for small list",
+InstallMethod( MultRowVectorLeft,
+    "Two argument kernel method for small list",
```

Comment:
"Two" -> "two"

But actually, more idiomatic would be to write "for a mutable dense small list and a multiplicative element" -- and this of course also applies to other `InstallMethod` calls you are adding or modifying.